src/radical/entk/execman/rp/task_manager.py

```diff
@@ -282,6 +286,17 @@ def load_placeholder(task):
                                                             {'path': task.path,
                                                              'uid': task.uid}
 
+        # ----------------------------------------------------------------------
+        def check_resource_reqs(task):
+
+            cpu_reqs = task.cpu_reqs['cpu_processes'] * task.cpu_reqs['cpu_threads']
+            gpu_reqs = task.gpu_reqs['gpu_processes'] * task.gpu_reqs['gpu_threads']
```

Comment:
RP currently interprets `gpu_processes` as `gpus_per_process`, and ignores `gpu_threads`.  So the above should ikely be:
```py
cpu_reqs = task.cpu_reqs['cpu_processes'] * task.cpu_reqs['cpu_threads']
gpu_reqs = task.cpu_reqs['cpu_processes'] * task.gpu_reqs['gpu_processes']
```

Reply:
I'll fix this and move forward with the rest. Thank you!